modules/ROOT/nav.adoc

```diff
@@ -155,12 +155,12 @@
  ** xref:analytics-event-api.adoc[Reviewing Analytics Event API]
  ** xref:analytics-event-forward.adoc[Reviewing Analytics Event Forwarding]
 * xref:mule-oauth-provider-landing-page.adoc[Mule OAuth 2.0 Provider]
- ** xref:oauth2-provider-configuration.adoc[Mule OAuth 2.0 Provider Configuration]
+ //** xref:oauth2-provider-configuration.adoc[Mule OAuth 2.0 Provider Configuration]
```

Comment:
Can we delete instead of comment out? To keep debt from accumulating in the file.